.circleci/config.yml

```diff
@@ -329,9 +329,11 @@ jobs:
             if [[ "<< parameters.k8sVersion >>" != "kind"* ]]; then
               export CI_K3S_VERSION=<< parameters.k8sVersion >>
               export K3D=true
+            else
+              export GINKGO_E2E_TEST_FLAGS="$GINKGO_E2E_TEST_FLAGS --label-filter=\"!kind-not-supported\""
             fi
             if [[ "<< parameters.arch >>" == "arm64" ]]; then
-              export GINKGO_E2E_TEST_FLAGS="--label-filter=\"!arm-not-supported\""
+              export GINKGO_E2E_TEST_FLAGS="$GINKGO_E2E_TEST_FLAGS --label-filter=\"!arm-not-supported\""
```

Comment:
I'd really like to have this and the other one in the makefile so that the CI config is as dumb as possible and we can run as much as we can locally. But when I did that I got [this](https://app.circleci.com/pipelines/github/kumahq/kuma/15514/workflows/1d92b0c3-6f48-4ccc-aae9-3b74954c5db2/jobs/190453?invite=true#step-110-3481) - anyone know why?

Reply:
Agreed that'd be nice it's weird it didn't work. Can you show the diff?

Reply:
https://github.com/kumahq/kuma/pull/4650/commits/87611c6b8de32b4fc2edeb2d622b689e4124b32a

Reply:
I know what was going on, I had quotes around that - I'll check if this works without that.